common/src/models/deployment.rs

```diff
@@ -163,13 +226,11 @@ pub fn get_deployments_table(
                 ]);
             } else {
                 table.add_row(vec![
-                    Cell::new(deploy.id),
+                    Cell::new(deploy.id.clone()),
                     Cell::new(&deploy.state)
                         // Unwrap is safe because Color::from_str returns the color white if str is not a Color.
                         .fg(Color::from_str(deploy.state.get_color()).unwrap())
                         .set_alignment(CellAlignment::Center),
-                    Cell::new(deploy.last_update.format("%Y-%m-%dT%H:%M:%SZ"))
-                        .set_alignment(CellAlignment::Center),
                     Cell::new(truncated_commit_id),
```

Comment:
we should avoid modifying old frontend's behaviour

Reply:
Done.